### PR TITLE
Travis integration

### PR DESCRIPTION
**Pull request by bmsbharathi**
Added AdoptJDK8 installation in Travis-CI integration.

### CODE REVIEW

**Comment by sclassen** [comment]
The updating and installing of the dependencies adds about 1 Minute to the build time.
Maybe using sdkman is faster.
We would need to check this in a separate branch

**Comment by bmsbharathi** [comment]
@sclassen hey! 
Just changed the JDK installation through sdkman.

File: AUTHORS [hunk 1]
```diff
@@ -3,6 +3,7 @@ Please keep this list in alphabetical order.
 
 Lillian Angel <langel@redhat.com>
 Andrew Azores <aazores@redhat.com>
+Bharathi BMS <bmsbharathi@gmail.com>
```
Comment (hendrikebbers) [inline AUTHORS:6]: While it is totally ok to add your name here the file is not really in sync anymore and I assume we will remove it in future :)

**Comment by hendrikebbers** [comment]
link to https://github.com/AdoptOpenJDK/IcedTea-Web/issues/700

**Comment by sclassen** [comment]
greate !!
SDK man brings a significant performance improvement over the deb packages.
Could you as a last step add the SDK man installation directory to the cache. Such that the JVM does not need to be re-downloaded from the internet.

**Comment by sclassen** [comment]
@bmsbharathi just want to quickly check back if you need any assistance with adding the SDK dir to the cache?

**Pull request merged**